src/systemtest/resources/ConfigSystemTest/report-config.yaml

```diff
@@ -15,15 +15,16 @@ group-details:
           - "**.md"
 repos:
   - repo: https://github.com/user/repo/tree/fake-branch
```

Comment:
This should also be the .git version of the URL. Same for src/test/resources/ReportConfigYamlParserTest/report-config-valid.yaml.